Library/Homebrew/cmd/install.rb

```diff
@@ -313,7 +339,7 @@ def run
           keep_tmp:                   args.keep_tmp?,
           debug_symbols:              args.debug_symbols?,
           force:                      args.force?,
-          overwrite:                  args.overwrite?,
+          overwrite:,
```

Comment:
This approach doesn't quite work, unfortunately. This is because the `--overwrite` flag doesn't propagate to dependencies.

Reply:
Passed it on to dependencies in 79959ca0d0b862347cfeade4eb024c50dbd545d2.

Reply:
> This approach doesn't quite work, unfortunately. This is because the `--overwrite` flag doesn't propagate to dependencies. Should it?

I don't think it should.